lib/http.js

```diff
@@ -43,7 +43,8 @@ HttpClient.prototype.buildRequest = function(rurl, data, exheaders, exoptions) {
     'Accept': 'text/html,application/xhtml+xml,application/xml,text/xml;q=0.9,*/*;q=0.8',
     'Accept-Encoding': 'none',
     'Accept-Charset': 'utf-8',
-    'Connection': 'close',
+    // Connection-Header is set in the http or https core module
+    // 'Connection': !!exoptions.forever ? 'keep-alive' : 'close',
```

Comment:
is this meant to be commented out?

Reply:
See: https://github.com/vpulim/node-soap/pull/979#issuecomment-335083728

Should I add "next line enables keep-alive connection only for pooled connections" to clearify that?

Reply:
I'd replace both lines with `'Connection': !!exoptions.forever ? 'keep-alive' : 'close',` and call it good.